client/src/pages/SearchUserPage.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import { Navigate } from "react-router-dom";
+import ErrorAlert from "../components/ErrorAlert";
+import {MOCK_USERS} from "../constants/mock/users";
+
+function SearchUserPage() {
+  let userView = () =>{
+    return MOCK_USERS.map((user)=>{
```

Comment:
nit: make this a variable instead of a function, since it's not dependent on any parameters
```jsx
const thing = ()=> {
  return <div>{x}</div>;
}
```
is the same as: 
```jsx
const thing = <div>{x}</div>;
```